Euclidean_Space.R

```diff
@@ -0,0 +1,83 @@
+#Euclidean_Space
+
+library(reticulate)
+gs <- import_from_path("geomstats", path = ".")
+
+EuclideanSpace <- setRefClass("EuclideanSpace",
+                              contains = Manifold,

```

Comment:
I think this should be `"Manifold"`. Let's get a simple test case running, this should confirm it.